src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MIN_LENGTH_OF_SIDE = 1;
+    private static final int MAX_LENGTH_OF_SIDE = 7;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int size = 5;
```

Comment:
with the current getRandomFigure() method implementation, its returns always the same figure. 

Switch-case takes variable size that always eq to 5.